src/modules/aws_account/entity/index.ts

```diff
@@ -2,32 +2,94 @@ import { Column, Entity, PrimaryGeneratedColumn, PrimaryColumn } from 'typeorm';
 
 import { cloudId } from '../../../services/cloud-id';
 
+/**
+ * Table that will hold the user's AWS credentials. When a new connection to IaSQL is issued,
+ * the AWS credentials are stored.
+ * The keys can be generated from the AWS console for each registered user
+ *
+ * @example
+ * ```sql
+ *  INSERT INTO aws_credentials (access_key_id, secret_access_key)
+ *  VALUES ('${process.env.AWS_ACCESS_KEY_ID}', '${process.env.AWS_SECRET_ACCESS_KEY}')
+ *
+ *  SELECT * FROM aws_credentials
+ * ```
+ *
+ * @see https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html
+ * @see https://github.com/iasql/iasql-engine/blob/main/test/modules/aws-account-integration.ts#L62
+ * @see https://github.com/iasql/iasql-engine/blob/main/test/modules/aws-account-integration.ts#L95
+ *
+ */
 @Entity()
 export class AwsCredentials {
+  /**
+   * @private
+   * @param
+   * Internal ID field for storing accounts
```

Comment:
Maybe:

```suggestion
   * Auto-incremented ID field for storing accounts
```

Reply:
It's not *exactly* internal since the user can access every column in the database, but it is set for them so they shouldn't need (or want) to do it themselves.

An "internal" field for us would be any properties from the AWS SDK that we use in the mappers but never expose to the user.